app/api/auth/create_access_token.go

```diff
@@ -91,16 +124,24 @@ func (srv *Service) createAccessToken(w http.ResponseWriter, r *http.Request) se
 		return apiError
 	}
 
-	// "Authorization" header is given, requesting a new token from the given token
-	if len(r.Header["Authorization"]) != 0 {
+	_, cookieErr := r.Cookie("access_token")
+
+	// "Authorization" header / "access_token" cookie is given, requesting a new token from the given token
+	if len(r.Header["Authorization"]) != 0 || cookieErr == nil {
 		if _, ok := requestData["code"]; ok {
 			return service.ErrInvalidRequest(
-				errors.New("only one of the 'code' parameter and the 'Authorization' header can be given"))
+				errors.New("only one of the 'code' parameter and the 'Authorization' header (or 'access_token' cookie) can be given"))
```

Comment:
Must be mentioned in the doc as well (probably with "use_cookie")

Reply:
fixed